CHANGELOG.md

```diff
@@ -30,6 +30,13 @@ You can check your current version with the following command:
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
+## 0.33.1
+
+**November 23, 2023**
```

Comment:
When we align on what should be included in the changelog, we also need to edit the banner in the `main.html` file.

Reply:
Agree :)